pom.xml

```diff
@@ -66,38 +67,21 @@
       <artifactId>mailer</artifactId>
       <version>1.8</version>
     </dependency>
+    <dependency> <!-- Can be removed after upgrade of parent version -->
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-core</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.sonatype.sisu</groupId>
+          <artifactId>sisu-guice</artifactId>
+        </exclusion>
+      </exclusions> 
+    </dependency>
     <dependency>
-      <groupId>com.google.code.findbugs</groupId>
-      <artifactId>annotations</artifactId>
-      <version>3.0.0</version>
-      <scope>provided</scope>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>4.0-beta</version>
```

Comment:
Almost :bug: I am not sure it is safe, because Jenkins core uses previous 3.x version. 4.0-beta has been introduced in 1.599 only: https://github.com/jenkinsci/jenkins/commit/a0942b88346d66770336daec7bf82063163e8001